salesforce-cdp/1.2/modules/ROOT/pages/salesforce-cdp-connector-config-topics.adoc

```diff
@@ -0,0 +1,52 @@
+= Salesforce Data Cloud - Additional Configuration Information - Mule 4
+
+Configure Anypoint Connector for Salesforce Data Cloud (Salesforce Data Cloud Connector) to generate an OAuth 2.0 keystore file.
+
+[[generate-keystore]]
+== Generate a Keystore File
+
+Salesforce Data Cloud Connector uses the Bouncy Castle cryptographic library to load the certificate from the keystore and sign the payload for the authentication requests.
+
+The Bouncy Castle library has a https://nvd.nist.gov/vuln/detail/CVE-2018-5382[vulnerability (CVE-2018-5382)] that relates to the BKS-V1 keystore file type, so do not use BKS-V1 keystore files, as documented in the https://www.kb.cert.org/vuls/id/306792[Bouncy Castle proposed solution]. By default, Salesforce Data Cloud Connector uses JKS or PKCS12 keystores.
+
+To generate a JKS keystore file:
+
+. Go to your Mule workspace and open the command prompt (for Windows) or Terminal (for Mac).
```

Comment:
Note that Windows 11 now calls the Powershell shortcut "Terminal" on the right-click menu and has a "Terminal" app (in addition to Powershell AND Command Prompt apps). I verified the command below works in ps as well, however you do get a warning that JKS is proprietary and PKCS12 should be used instead.

Reply:
@kckane-mulesoft I don't have a Windows machine. Can we just say something generic like "open the command line app for your OS? Or something like that?

Reply:
Sure, or maybe "terminal/shell application".